---
title: Publications
date: 2021-09-08
tags: 
- pubs
---



# Publications

80 Weishuai Di, Kai Xue, Jun Cai, Zhenshu Zhu, Zihan Li, Hui Fu, Hai Lei, Wenbing Hu, Chun Tang, Wei Wang, and Yi Cao, "Single-Molecule Force Spectroscopy Reveals Cation-π Interactions in Aqueous Media Are Highly Affected by Cation Dehydration" Phys. Rev. Lett. 130, 118101 [https://doi.org/10.1103/PhysRevLett.130.118101](https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.130.118101)

79 Weitao Fu, Hao Yang, Chenxian Hu, Jianing Liao, Zhou Gong, Minkui Zhang, Shuai Yang, Shangxiang Ye, Yixuan Lei, Rong Sheng, Zhiguo Zhang, Xiaojun Yao, Chun Tang*, Dan Li*, and Tingjun Hou*, "Small-Molecule Inhibition of Androgen Receptor Dimerization as a Strategy against Prostate Cancer" ACS Cent. Sci. [https://doi.org/10.1021/acscentsci.2c01548](https://pubs.acs.org/doi/full/10.1021/acscentsci.2c01548)

78 	Wang Jian-Hua, Gong Zhou, Dong Xu, Liu Shu-Qun, Tang Yu-Liang, Lei Xiaoguang, Tang Chun, Dong Meng-Qiu, "Fast cross-linking by DOPA2 promotes the capturing of a stereospecific protein complex over nonspecific encounter complexes". Biophysics Reports. [https://doi.org/10.52601/bpr.2022.220014](http://www.biophysics-reports.org/en/article/doi/10.52601/bpr.2022.220014) 

77 Xuhui Yan, Kai Pei, Zeyuan Guan, Feiqing Liu, Junjun Yan, Xiaohuan Jin, Qiang Wang, Mengjun Hou, Chun Tang & Ping Yin, "AI-empowered integrative structural charac terization of m<sup>6</sup>A methyltransferase complex" Cell Research (2022) [https://doi.org/10.1038/s41422-022-00741-8](https://www.nature.com/articles/s41422-022-00741-8)

76 Jian-Hua Wang, Yu-Liang Tang, Zhou Gong, Rohit Jain, Fan Xiao, Yu Zhou, Dan Tan, Qiang Li, Niu Huang, Shu-Qun Liu, Keqiong Ye, Chun Tang, Meng-Qiu Dong & Xiaoguang Lei, "Characterization of protein unfolding by fast cross-linking mass spectrometry using di-ortho-phthalaldehyde cross-linkers" Nat Commun 13, 1468 (2022). [https://doi.org/10.1038/s41467-022-28879-4](https://www.nature.com/articles/s41467-022-28879-4#citeas)

75 Yingxue Ma, Haozheng Li, Zhou Gong, Shuai Yang, Ping Wang, and Chun Tang, **2022**,"Nucleobase Clustering Contributes to the Formation and Hollowing of Repeat-Expansion RNA Condensate" J. Am. Chem. Soc. [https://doi.org/10.1021/jacs.1c12085](https://pubs.acs.org/doi/10.1021/jacs.1c12085)

74 Xu Dong, Ling-Yun Qin, Zhou Gong, Sanbo Qin, Huan-Xiang Zhou, and Chun Tang,**2022**."Preferential Interactions of a Crowder Protein with the Specific Binding Site of a Native Protein Complex" J. Phys. Chem. Lett.: 792–800[https://doi.org/10.1021/acs.jpclett.1c03794](https://pubs.acs.org/doi/full/10.1021/acs.jpclett.1c03794)


73 Czaplewski, C., et al. **(2021)**. "Recent Developments in Data-Assisted Modeling of Flexible Proteins." Frontiers in Molecular Biosciences 8(1306).[https://doi.org/10.3389/fmolb.2021.765562](https://www.frontiersin.org/articles/10.3389/fmolb.2021.765562/full)


72 Fu, W., et al. **(2021)**. "Discovery of a Novel Androgen Receptor Antagonist Manifesting Evidence to Disrupt the Dimerization of the Ligand-Binding Domain via Attenuating the Hydrogen-Bonding Network Between the Two Monomers." Journal of Medicinal Chemistry 64(23): 17221-17238.[https://doi.org/10.1021/acs.jmedchem.1c01287](https://pubs.acs.org/doi/abs/10.1021/acs.jmedchem.1c01287)

71 Kogut M, Gong Z, Tang C, Liwo A. **(2021)** Pseudopotentials for coarse‐grained cross‐link‐assisted modeling of protein structures
M Kogut, Z Gong, C Tang, A Liwo Journal of Computational Chemistry 42 (29), 2054-2067[ *Journal of Computational Chemistry,* doi.org/10.1002/jcc.26736](http://onlinelibrary.wiley.com/doi/10.1002/jcc.26736)

<!--more-->

70 Qin LY, Gong Z, Liu K, Dong X and Tang C. **(2021)** Kinetic Constraints in the Specific Interaction between Phosphorylated Ubiquitin and Proteasomal Shuttle Factors. [*Biomolecules*](https://www.mdpi.com/2218-273X/11/7/1008)[, 10.3390/biom11071008](https://www.mdpi.com/2218-273X/11/7/1008)*[
](https://www.mdpi.com/2218-273X/11/7/1008)*

69 Lerner E, Barth A, Hendrix J, Ambrose B, Birkedal V, Blanchard SC, Börner R,Chung HS, Cordes T, Craggs TD, Deniz AA, Diao JJ, Fei JY, Gonzalez RL, Gopich IV, Ha T, Hanke CA, Haran G, Hatzakis NS, Hohng SC, Hong SC, Hugel T, Ingargiola A, Joo C, Kapanidis AN, Kim HD, Laurence T, Lee NK, Lee TH, Lemke EA, Margeat E, Michaelis J, Michalet X, Myong S, Nettels D, Peulen TO, Ploetz E, Razvag Y, Robb NC, Schuler B, Soleimaninejad H, Tang C, Vafabakhsh R, Lamb DC, Seidel CA, Weiss S. **(2021)** FRET-based dynamic structural biology: Challenges, perspectives and an appeal for open-science practices. *[eLife 2021;10:e60416](https://elifesciences.org/articles/60416)*

68 Gong Z, Ye SX and Tang C. **(2020)** Tightening the crosslinking distance restraints for better resolution of protein structure and dynamics.*[Structure](https://www.cell.com/structure/fulltext/S0969-2126(20)30243-4?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0969212620302434%3Fshowall%3Dtrue)*[, 28:1160-1167](https://www.cell.com/structure/fulltext/S0969-2126(20)30243-4?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0969212620302434%3Fshowall%3Dtrue)

67 Yang J, Gong Z, Lu YB, Xu CJ, Wei TF, Yang MS, Zhan TW, Yang YH, Lin L, Liu JF, Tang C and Zhang WP. **(2020)** FLIM-FRET based structural characterization of a class-A GPCR dimer in the cell membrane. [*J Mol Biol*, 432:4596-4611](https://www.sciencedirect.com/science/article/pii/S0022283620304058?via%3Dihub)

66 Gong Z, Yang S, Dong X, Yang QF, Zhu YL, Xiao Y, Tang C. **(2020)** Hierarchical conformational dynamics confers thermal adaptability to preQ1 RNA riboswitches.[ *J Mol Biol**,* 432:4523-4543](https://www.sciencedirect.com/science/article/pii/S0022283620303892?via%3Dihub)

65 Gong Z, Ye SX, Nie ZF and Tang C. **(2020)** The conformational preference of chemical crosslinkers determins the crosslinking probability of reactive protein residues. *[J](https://pubs.acs.org/doi/10.1021/acs.jpcb.0c02522)[ Phy Chem B](https://pubs.acs.org/doi/10.1021/acs.jpcb.0c02522)*[, 123:4446-4453](https://pubs.acs.org/doi/10.1021/acs.jpcb.0c02522)

64 Tang C and Gong Z. **(2020)** Integrating Non-NMR Distance Restraints to Augment NMR Depiction of Protein Structure and Dynamics.[ *J Mol Biol*, 432:2913-2929](https://www.sciencedirect.com/science/article/pii/S0022283620300875?via%3Dihub)

63 Tang C and Zhang WP. **(2020)** How phosphorylation by PINK1 remodels the ubiquitin system, a perspective from structure and dynamics. [*Biochemistry*](https://pubs.acs.org/doi/10.1021/acs.biochem.9b00715?ai=7743), 59:26-33

62 You ZY, Jiang WX, Qin LY, Gong Z, Wan W, Li J, Wang YS, Zhang HT, Peng C, Zhou TH, Tang C and Liu W. **(2019)** Requirement for p62 acetylation in the aggregation of ubiquitylated proteins under nutrient stress*.**[Nat Commun](https://www.nature.com/articles/s41467-019-13718-w),* 10:5792

61 Gong Z, Yang S, Yang QF, Zhu YL, Jiang J and Tang C. **(2019)** Refining RNA solution structures with the integrative use of label-free paramagnetic relaxation enhancement NMR. [*Biophys. Rep*](https://link.springer.com/article/10.1007/s41048-019-00099-2), 5:244-253

60 Ye SX, Gong Z, Yang J, An YX, Liu Z, Zhao Q, Lescop E, Dong X, Tang C. **(2019)** Ubiquitin is double-phosphorylated by PINK1 for enhanced pH-sensitivity of conformational switch. [*Protein & Cell*](https://link.springer.com/article/10.1007%2Fs13238-019-0644-x), 10:908-913

59 Yang QF, Tang C. **(2019)** On the necessity of an integrative approach to understand protein structural dynamics.[ *J Zhejiang Univ Sci B*](https://link.springer.com/article/10.1631/jzus.B1900135), 20(6):496-502

58 Liu Z, Dong X, Yi HW, Yang J, Gong Z, Wang Y, Liu K, Zhang WP, Tang C. **(2019)** Structural basis for the recognition of K48-linked Ub chain by proteasomal receptor Rpn13. [*Cell Discov*, 5:19](https://www.nature.com/articles/s41421-019-0089-7)

57 Huang JB, Dong X, Gong Z, Qin LY, Yang S, Zhu YL, Wang X, Zhang DL, Zou TT, Yin P and Tang C. **(2019)** Solution structure of the RNA recgonition domain of METTL3-METTL14 N6-methyladenosine methyltransferase. [*Protein & Cell*, 10:272-284](https://link.springer.com/content/pdf/10.1007%2Fs13238-018-0518-7.pdf)

56 Gong Z, Charles DS and Tang C **(2018)** Theory and practice of using solvent paramagnetic relaxtion enhancement to characterize protein conformational dynamics. [*Methods*, 148:48-56](https://www.sciencedirect.com/science/article/pii/S1046202318300094)

55 Zhang XY, Wang JH, Tan D, Li Q, Li MD, Gong Z, Tang C, Liu ZL, Dong MQ, Lei XG. **(2018).** Carboxylate-selective chemical cross-linkers for mass spectrometric analysis of protein structures. [*Anal Chem, 90: 1195-1201*](http://pubs.acs.org/doi/abs/10.1021/acs.analchem.7b03789)

54 Liu Z, Gong Z, Cao Y, Ding YH, Dong MQ, Lu YB, Zhang WP, Tang C. **(2018)**.Characterizing Protein Dynamics with Integrative Use of Bulk and Single-Molecule Techniques. [*Biochemistry, 57:305-313*](http://pubs.acs.org/doi/10.1021/acs.biochem.7b00817)

53 Gong Z, Liu Z, Dong X, Ding YH, Dong MQ, Tang C. **(2017)**. Protocol for analyzing protein ensemble structures from chemical cross-links using DynaXL. *[Biophys. Rep. 3, 100-108](http://www.biophysics-reports.org/EN/10.1007/s41048-017-0044-9)*

52 Dong X, Gong Z, Lu YB, Liu K, Qing LY, Ran ML, Zhang CL, Liu Z, Zhang WP, Tang C. **(2017).** Ubiquitin S65 phosphorylation engenders a pH-sensitive conformational switch. [*Proc Natl Acad Sci, USA*, ](http://www.pnas.org/content/early/2017/06/12/1705718114.abstract)*114:6770-6775*

51 Ge Fang, Binquan Luan, Cuicui Ge, Yu Chong, Xu Dong, Jun Guo, Chun Tang, Ruhong Zhou. **(2017)**. Understanding the graphene quantum dots-ubiquitin interaction by identifying the interaction sites. [*Carbon*, ](http://www.sciencedirect.com/science/article/pii/S0008622317305535)*112:285-291*

50 Jiang WX, Gu XH,Dong X, Tang C. **(2017)**. Lanthanoid tagging via an unnatural amino acid for protein structure characterization.[ *J Biomol NMR*,67:273](https://link.springer.com/article/10.1007%2Fs10858-017-0106-9)-282[
](https://link.springer.com/article/10.1007%2Fs10858-017-0106-9)

49 Guo XD, Yan CS, Li H, Huang WM, Shi XS, Huang M, Yang YF, Pan WL, Cai MJ, Li LY, Wu W, Bai YB, Zhang C, Liu ZJ, Wang XY, Zhang XH, Tang C, Wang HD, Liu WL, Wong CC, Cao Y, Xu CQ. **(2017)**. Lipid-dependent conformational dynamics underlie the functional versatility of T-cell receptor. [*Cell Research*, 27,505-525.](https://www.nature.com/cr/journal/v27/n4/full/cr201742a.html)

48 Ding YH, Gong Z, Dong X, Liu K, Liu Z, Liu C, He SM, Dong MQ, Tang C. **(2017)**. Modeling protein excited-state structures from "over-length" chemical cross-links.[ *J Biol Chem*, ](http://www.jbc.org/content/early/2016/12/19/jbc.M116.761841.abstract?ctkey=spe#xref-corresp-1-1)292,1187-1196

47 Gong Z, Gu XH, Guo DC, Wang J, Tang C. **(2017)**. Protein structural ensembles visualized by solvent paramagnetic relaxation enhancement. [*Angew Chem Int Ed*, 56,1002-1006](http://onlinelibrary.wiley.com/doi/10.1002/anie.201609830/full)

46 Zhang JR, Liu N, Chacho RA, Gong Z, Liu Z, Qin WM, Tang C, Tang Y, Zhou JH. **(2016)**.  Structural basis of nonribosomal pepetide macrocyclization in fungi. [*Nat Chem Biol*, ](http://www.nature.com/nchembio/journal/vaop/ncurrent/full/nchembio.2202.html)12, 1001-1003

45 Wang X, Feng J, Xue Y, Guan ZY, Zhang DL, Liu Z, Gong Z, Wang Q, Huang JB, Tang C, Zou TT, Yin P. **(2016)** Structural basis of N6-adenosine methylation by the METTL3–METTL14 complex. [*Nature,* 534,575-578.](http://dx.doi.org/10.1038/nature18298)

44 Fang J, Cheng J, Wang J, Zhang Q, Liu M, Gong R, Wang P, Zhang X, Feng Y, Lan W, Gong Z, Tang C, Wong J, Yang H, Cao C, Xu Y. **(2016)** Hemi-methylated DNA opens a closed conformation of UHRF1 to facilitate its histone recognition. [*Nature Communications,* doi:10.1038/ncomms11197.](http://www.nature.com/ncomms/2016/160405/ncomms11197/full/ncomms11197.html)

43 Jiang WX, Dong X, Jiang J, Yang YH, Yang J, Lu Yb, Fang SH, Wei EQ, Tang C, and Zhang WP. **(2016)** Specific cell surface labeling of GPCRs using split GFP. [*Sci.Rep.* 6:20568](https://www.nature.com/articles/srep20568)

42 Liu Z and Tang C. **(2016)** Ensemble structure description of Lys63-linked diubiquitin. [*Data in Brief,* doi:10.1016/j.dib.2016.02.003.](http://www.sciencedirect.com/science/article/pii/S2352340916300257)

41 Gong Z, Ding YH, Dong X, Liu N, Zhang EE, Dong MQ and Tang C. **(2015)** Visualizing the ensemble structures of protein complexes using chemical cross-linking coupled with mass spectrometry. *[Biophys.Rep.](http://link.springer.com/article/10.1007%2Fs41048-015-0015-y), 1, 127-138*

40 Liu Z, Gong Z, Dong X and Tang C. **(2016)** Transient protein-protein interactions visualized by solution NMR. [*BBA Proteins and Proteomics,* 1864, 115-122](http://www.sciencedirect.com/science/article/pii/S1570963915001065). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/25896389) **
**

39 Liu Z, Gong Z, Jiang WX, Yang J, Zhu WK, Guo DC, Zhang WP, Liu ML and Tang C. **(2015)** Lys63-linked ubiquitin chain adopts multiple conformational states for specific target recognition. *[eLife](http://elifesciences.org/content/early/2015/06/19/eLife.05767)*[,4:e05767](http://elifesciences.org/content/4/e05767?utm_source=content_alert&utm_medium=email&utm_content=fulltext&utm_campaign=elife-alerts). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/26090905)

38 Zhang XQ, Lu JT, Jiang WX, Lu YB, Wu M, Wei EQ, Zhang WP and Tang C. **(2015)** NAMPT inhibitor and metabolite protect mouse brain from cryoinjury through distinct mechanisms. [*Neuroscience,* 291*,* ](http://www.sciencedirect.com/science/article/pii/S0306452215001499)[230-40](http://www.sciencedirect.com/science/article/pii/S0306452215001499). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/25684751)

37 Gong Z, Charles DS and Tang C. **(2015)** Conjoined use of EM and NMR in RNA structure refinement. [*Plos One,* 10, e0120445](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0120445). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/25798848)

36 Xing Q, Huang P, Yang J, Sun JQ, Gong Z, Dong X, Guo DC, Chen SM, Yang YH, Wang Y, Yang MH, Yi M, Ding YM, Liu ML, Zhang WP and Tang C. **(2014)** Visualizing an ultra-weak protein-protein interaction in phosphorylation signaling. [*Angew Chem Int Ed* ,53](http://onlinelibrary.wiley.com/doi/10.1002/anie.201405976/abstract;jsessionid=4239070CBF7D7017CFC1F6528F2BC77A.f03t02),[ 11501-11505](http://onlinelibrary.wiley.com/doi/10.1002/anie.201405976/abstract;jsessionid=4239070CBF7D7017CFC1F6528F2BC77A.f03t02). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=25131700) **(back cover)**

35 Wang Y, Tang C, Wang EK and Wang J. **(2014)** Polyubiquitin chain linkage topology selects the functions from the underlying binding landscape. *[Plos Comput Biol](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003691),* [10](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003691). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=24992446)

34 Zhang Z, Zhang TL, Wang SS, Gong Z, Tang C, Chen JY and Ding JP. **(2014)** Molecular mechanism for Rabex-5 GEF activation by Rabaptin-5. [*eLife,* doi:10.7554/eLife.02687](http://elifesciences.org/content/3/e02687)*.* [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/24957337)

33 Rathinavelan T, Lara-Tejero M, Lefebre M, Chatterjee S, McShan AC, Guo DC, Tang C, Galan JE and De Guzman RN. **(2014)** NMR model of prgI-sipD interaction and its implications in the needle-tip assembly of the salmonella type III secretion system. [*J Mol Biol* 426](http://www.sciencedirect.com/science/article/pii/S0022283614003027),[ 2958-2969](http://www.sciencedirect.com/science/article/pii/S0022283614003027). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/24951833)

32 Liu Z, Gong Z, Guo DC, Zhang WP and Tang C. **(2014)** Subtle dynamics of holo glutamine binding protein revealed with a rigid paramagnetic probe. [*Biochemistry-Us,* 53, 1403-1409](http://pubs.acs.org/doi/abs/10.1021/bi4015715). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/24555491)

31 Gu XH, Gong Z, Guo DC, Zhang WP and Tang C. **(2014)** A decadentate Gd(III)-coordinating paramagnetic cosolvent for protein relaxation enhancement measurement. [*J Biomol N**MR,* 58, 149-154](http://link.springer.com/article/10.1007%2Fs10858-014-9817-3). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/24510274)

30 Coffman K, Yang B, Lu J, Tetlow AL, Pelliccio E, Lu S, Guo DC, Tang C, Dong MQ and Tamanoi F. **(2014)** Characterization of the Raptor/ 4E-BP1 Interaction by chemical cross- linking coupled with mass spectrometry analysis. [*J Biol Chem*, 289, 4723-4734](http://www.jbc.org/content/289/8/4723.long). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/24403073)

29 Zhao B, Zhang M, Han X, Zhang XY, Xing Q, Dong X, Shi QJ, Huang P, Lu YB, Wei EQ, Xia Q, Zhang WP and Tang C. **(2013)** Cerebral ischemia is exacerbated by extracellular nicotinamide phosphoribosyltransferase via a non-enzymatic mechanism. [*Plos One,* 8](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0085403). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=24392007)

28 Li CG, Tang C and Liu ML. **(2013)** Protein dynamics elucidated by NMR technique. [*Protein Cell,* 4, 726-730](http://link.springer.com/article/10.1007%2Fs13238-013-3912-1). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=24104391)

27 Jiang B, Luo F, Ding YM, Sun P, Zhang X, Jiang L, Li CG, Mao XA, Yang DW, Tang C and Liu ML. **(2013)** NASR: an effective approach for simultaneous noise and artifact suppression in NMR spectroscopy. [*Anal Chem,* 85, 2523-2528](http://pubs.acs.org/doi/abs/10.1021/ac303726p). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=23339579)

26 Wang Y, Tang C, Wang EK and Wang J. **(2012)** Exploration of multi-state conformational dynamics and underlying global functional landscape of maltose binding protein. [*Plos Comput Biol,*8](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1002471). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=22532792)

25 Liu Z, Zhang WP, Xing Q, Ren XF, Liu ML and Tang C. **(2012)** Noncovalent dimerization of ubiquitin. [*Angew Chem Int Ed,* 51, 469-472](http://onlinelibrary.wiley.com/doi/10.1002/anie.201106190/abstract). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=22109817)

24 刘主，唐淳. **(2011)** 顺磁弛豫增强技术与蛋白质瞬态结构. [*波谱学杂志,* 28, 301-316](http://magres.wipm.ac.cn/CN/abstract/abstract9455.shtml).

23 Rathinavelan T, Tang C and De Guzman RN. **(2011)** Characterization of the interaction between the salmonella type III secretion system tip protein sipD and the needle protein prgI by paramagnetic relaxation enhancement. [*J Biol Chem,* 286, 4922-4930](http://www.jbc.org/content/286/6/4922.long). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=21138848)

22 Yu DM, Volkov AN and Tang C. **(2009)** Characterizing dynamic protein-protein interactions using differentially scaled paramagnetic relaxation enhancement. [*J Am Chem Soc,* 131, 17291-17297](http://pubs.acs.org/doi/abs/10.1021/ja906673c). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=19891483)

**Before 2009**

21 Tang C, Louis JM, Aniana A, Suh JY and Clore GM. **(2008)** Visualizing transient events in amino-terminal autoprocessing of HIV-1 protease. [*Nature,* 455, 693-696](http://www.nature.com/nature/journal/v455/n7213/full/nature07342.html). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=18833280)

20 Tang C, Ghirlando R and Clore GM. **(2008)** Visualization of transient ultra-weak protein self-association in solution using paramagnetic relaxation enhancement. [*J Am Chem Soc,* 130, 4048-4056](http://pubs.acs.org/doi/abs/10.1021/ja710493m). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=18314985)

19 Kim YC, Tang C, Clore GM and Hummer G. **(2008)** Replica exchange simulations of transient encounter complexes in protein-protein association. [*P Natl Acad Sci USA,* 105, 12855-12860](http://www.pnas.org/content/105/35/12855.long). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=18728193)

18 Tang C, Schwieters CD and Clore GM. **(2007)** Open-to-closed transition in apo maltose-binding protein observed by paramagnetic NMR. [*Nature* 449, 1078-1082](http://www.nature.com/nature/journal/v449/n7165/full/nature06232.html). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=17960247)

17 Clore GM, Tang C and Iwahara J. **(2007)** Elucidating transient macromolecular interactions using paramagnetic relaxation enhancement. [*Curr Opin Struc Biol,* 17, 603-616](http://www.sciencedirect.com/science/article/pii/S0959440X07001200). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=17913493)

16 Iwahara J, Tang C and Marius Clore G. **(2007)** Practical aspects of (1)H transverse paramagnetic relaxation enhancement measurements on macromolecules. [*Journal of magnetic resonance,* 184, 185-195](http://www.sciencedirect.com/science/article/pii/S109078070600334X). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=17084097)

15 Kelly BN, Kyere S, Kinde I, Tang C, Howard BR, Robinson H, Sundquist WI, Summers MF and Hill CP. **(2007)** Structure of the antiviral assembly inhibitor CAP-1 complex with the HIV-1 CA protein. [*J Mol Biol,* 373, 355-366](http://www.sciencedirect.com/science/article/pii/S0022283607010492).[ [pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=17826792)

14 Suh JY, Tang C and Clore GM. **(2007)** Role of electrostatic interactions in transient encounter complexes in protein-protein association investigated by paramagnetic relaxation enhancement. [*J Am Chem Soc,* 129, 12954-12955](http://pubs.acs.org/doi/abs/10.1021/ja0760978). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=17918946)

13 Tang C, Iwahara J and Clore GM. **(2006)** Visualization of transient encounter complexes in protein-protein association. [*Nature,* 444, 383-386](http://www.nature.com/nature/journal/v444/n7117/full/nature05201.html). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=17051159)

12 Tang C and Clore GM. **(2006)** A simple and reliable approach to docking protein-protein complexes from very sparse NOE-derived intermolecular distance restraints. [*J Biomol N**MR,* 36, 37-44](http://link.springer.com/article/10.1007%2Fs10858-006-9065-2). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=16967193)

11 Tang C, Williams DC, Ghirlando R and Clore GM. **(2005)** Solution structure of enzyme IIA(Chitobiose) from the N,N '-diacetylchitobiose branch of the Escherichia coli phosphotransferase system. [*J Biol Chem* 280, 11770-11780](http://www.jbc.org/content/280/12/11770.long). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=15654077)

10 Suh JY, Tang C, Cai ML and Clore GM. **(2005)** Visualization of the phosphorylated active site loop of the cytoplasmic B domain of the mannitol transporter IIMannitol of the Escherichia coli phosphotransferase system by NMR spectroscopy and residual dipolar couplings. [*J Mol Biol* 353, 1129-1136](http://www.sciencedirect.com/science/article/pii/S0022283605010983). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=16219324)

9 Tang C, Iwahara J and Clore G. **(2005)** Accurate determination of leucine and valine side-chain conformations using U-[N-15/C-13/H-2]/[H-1-(methine/methyl)-Leu/Val] isotope labeling, NOE pattern recognition, and methine C gamma-H gamma/C beta-H beta residual dipolar couplings: application to the 34-kDa enzyme IIA (Chitobiose). [*J Biomol N**MR* 33, 105-121](http://link.springer.com/article/10.1007%2Fs10858-005-1206-5). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=16258829)

8 Tang C, Loeliger E, Luncsford P, Kinde I, Beckett, D and Summers MF. **(2004)** Entropic switch regulates myristate exposure in the HIV-1 matrix protein. [*P Natl Acad Sci USA* 101, 517-522](http://www.pnas.org/content/101/2/517.long). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=14699046)

7 Tang C, Loeliger E, Kinde I, Kyere S, Mayo K, Barklis E, Sun YN, Huang MJ and Summers MF. **(2003)** Antiviral inhibition of the HIV-1 capsid protein. [*J Mol Biol* 327, 1013-1020](http://www.sciencedirect.com/science/article/pii/S0022283603002894). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=12662926)

6 Gatanaga H, Suzuki Y, Tsang H, Yoshimura K, Kavlick MF, Nagashima K, Gorelick R. J, Mardy S, Tang C, Summers MF and Mitsuya H. **(2002)** Amino acid substitutions in Gag protein at non-cleavage sites are indispensable for the development of a high multitude of HIV-1 resistance against protease inhibitors.[ *J Biol Chem* 277, 5952-5961](http://www.jbc.org/content/277/8/5952.long). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=11741936)

5 Tang C, Ndassa Y and Summers MF. **(2002)** Structure of the N-terminal 283-residue fragment of the immature HIV-1 Gag polyprotein. [*Nat Struct Biol* 9, 537-543](http://www.nature.com/nsmb/journal/v9/n7/full/nsb806.html). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=12032547)

4 Lin J, Blake M, Tang C, Zimmer D, Rustandi RR, Weber DJ and Carrier F. **(2001)** Inhibition of p53 transcriptional activity by the S100B calcium-binding protein. [*J Biol Chem,* 276, 35037-35041](http://www.jbc.org/content/276/37/35037.long). [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=11454863)

3 Gu T, Tang C and Xu ZD. **(1999)** Synthesis of C-60 nitroxide derivatives. [*Chinese Chem Lett,* 10,95-96.](http://www.jbc.org/content/276/37/35037.long)

2 Gu T, Tang, C and Xu ZD. **(1999)** Two methods to synthesize C-60 nitroxide derivatives. [*Fullerence Sci Techn,* 7,297-303.](http://www.jbc.org/content/276/37/35037.long)

1 Baldisseri DM, Rustandi RR, Zhang ZS, Tang C, Bair C. L, Landar A, Landar A, Zimmer DB and Weber DJ. **(1999)** Letter to the Editor: H-1, C-13 and N-15 NMR sequence-specific resonance assignments for rat apo-S100A1(alpha alpha).*[J Biomol NMR, ](http://www.jbc.org/content/276/37/35037.long)*[14,91-92](http://www.jbc.org/content/276/37/35037.long) *.* [[pubmed\]](http://www.ncbi.nlm.nih.gov/pubmed/?term=10382312)
