---
title: "Principal investigator"
description: "Principal investigator"
lead: "唐淳"
date: 2024-09-02
lastmod: 2020-11-12T15:22:20+01:00
draft: false
images: []
tags:
- memb
- members
menu: 
  member:
    parent: "group-leader"
weight: 10
toc: false
top: true
no_word_count: true
no_toc: true
---

![唐淳](https://raw.githubusercontent.com/DF-Master/yidapicbed/main/markdown/20210902103406.jpg)

E-mail: Tang_Chun@pku.edu.cn

Website: http://www.tanglab.org

<!--more-->

## Research interests:

* Method development of paramagnetic NMR
* Method development for integrative structural biology refinement, to better utilize the data from small angle scattering, single-molecular FRET, and chemical crosslinking, etc.
* To understand how post-translational/transcriptional modifications impact the structure and dynamics of proteins and RNAs, with an emphasis on ubiquitin phosphorylation and RNA methylation

## Selected publications:

1. Tang C, Gong Z. (2020) Integrating Non-NMR Distance restraints to augment NMR depiction of protein structure and dynamics. J Mol Biol, doi: 10.1016/j.jmb.2020.01.023
2. Tang C, Zhang WP. (2020) How phosphorylation by PINK1 remodels the ubiquitin system, a perspective from structure and dynamics. Biochemistry, 59:26-33
3. You ZY, Jiang WX, Qin LY, Gong Z, Wan W, Li J, Wang YS, Zhang HT, Peng C, Zhou TH, Tang C, Liu W. (2019) Requirement for p62 acetylation in the aggregation of ubiquitylated proteins under nutrient stress. Nat Commun, 10:5792
4. Huang JB, Dong X, Gong Z, Qin LY, Yang S, Zhu YL, Wang X, Zhang DL, Zou TT, Yin P, Tang C. (2019) Solution structure of the RNA recognition domain of METTL3-METTL14 N6-methyladenosine methyltransferase. Protein Cell, 10:272-284
5. Gong Z, Yang S, Yang QF, Zhu YL, Jiang J, Tang C. (2019) Refining RNA solution structures with the integrative use of label-free paramagnetic relaxation enhancement NMR. Biophys Rep, 5:244-253
6. Gong Z, Charles DS, Tang C (2018) Theory and practice of using solvent paramagnetic relaxation enhancement to characterize protein conformational dynamics. Methods, 148:48-56
7. Liu Z, Gong Z, Cao Y, Ding YH, Dong MQ, Lu YB, Zhang WP, Tang C. (2018). Characterizing protein dynamics with integrative use of bulk and single-molecule techniques. Biochemistry, 57:305-313
8. Ding YH, Gong Z, Dong X, Liu K, Liu Z, Liu C, He SM, Dong MQ, Tang C. (2017). Modeling protein excited-state structures from "over-length" chemical cross-links. J Biol Chem, 292:1187-1196
9. Jiang WX, Gu XH, Dong X, Tang C. (2017). Lanthanoid tagging via an unnatural amino acid for protein structure characterization. J Biomol NMR, 67:273-282
10. Dong X, Gong Z, Lu YB, Liu K, Qing LY, Ran ML, Zhang CL, Liu Z, Zhang WP, Tang C. (2017). Ubiquitin S65 phosphorylation engenders a pH-sensitive conformational switch. Proc Natl Acad Sci, USA, 114:6770-6775
11. Gong Z, Gu XH, Guo DC, Wang J, Tang C. (2017). Protein structural ensembles visualized by solvent paramagnetic relaxation enhancement. Angew Chem Int Ed, 56:1002-1006
12. Liu Z, Gong Z, Dong X, Tang C. (2016) Transient protein-protein interactions visualized by solution NMR. BBA Proteins and Proteomics, 1864:115-122
13. Liu Z, Gong Z, Jiang WX, Yang J, Zhu WK, Guo DC, Zhang WP, Liu ML, Tang C. (2015) Lys63-linked ubiquitin chain adopts multiple conformational states for specific target recognition. eLife, 4:e05767
14. Gong Z, Charles DS, Tang C. (2015) Conjoined use of EM and NMR in RNA structure refinement. Plos One, 10, e0120445
15. Xing Q, Huang P, Yang J, Sun JQ, Gong Z, Dong X, Guo DC, Chen SM, Yang YH, Wang Y, Yang MH, Yi M, Ding YM, Liu ML, Zhang WP, Tang C. (2014) Visualizing an ultra-weak protein-protein interaction in phosphorylation signaling. Angew Chem Int Ed, 53:11501-11505